.cargo/config.toml

```diff
@@ -0,0 +1,6 @@
+# Find the full list of available options here: https://doc.rust-lang.org/cargo/reference/config.html
+
+[alias]
+acfmt = "fmt --all -- --check"
+aclippy = "clippy --workspace --all-targets --all-features -- -D warnings"
+atest = "test --workspace --all-targets --all-features "
```

Comment:
words are cheap, I suggest we use longer aliases just the sake of clarity? 

```
fmt-amaru
clippy-amaru
test-amaru
```

?

Reply:
Sounds better!